src/main/java/com/hedvig/contract/shared/events/PendingAgreementActivated.kt

```diff
@@ -1,9 +1,9 @@
 package com.hedvig.contract.shared.events
 
-import com.hedvig.contract.shared.aggregates.BaseAgreement
+import com.hedvig.contract.shared.stereotypes.AgreementStereotype
 import java.util.UUID
 
-class PendingAgreementActivated<TAgreement : BaseAgreement>(
+class PendingAgreementActivated<TAgreement : AgreementStereotype>(
```

Comment:
This should not be changed. This way there could in theory exist an event with SwedishApartmentAgreementState which should not be allowed.

```suggestion
class PendingAgreementActivated<TAgreement : BaseAgreement>(
```